package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.0.1",
   "description": "128 and 256 bits integer and fixed point arithmetics. Also support checking overflow/underflow",
   "main": "js/index.js",
+  "types": "assembly/index.ts",
   "author": "MaxGraey <maxgraey@gmail.com>",
   "license": "Apache-2.0",
   "scripts": {
     "build:debug": "npx asc assembly/index.ts -b build/debug/bignum.wasm -t build/debug/bignum.wat --sourceMap --validate --debug",
     "build:release": "npx asc assembly/index.ts -b build/release/bignum.wasm -t build/release/bignum.wat -O3 --sourceMap --validate",
```

Comment:
And here

Reply:
for "build:release" plz as well

Reply:
Ah okay thought releases should use full.